bin/sl-pmctl.js

```diff
@@ -432,9 +432,16 @@ function extra() {
 
 function remoteRequest(pmctl, cmd, callback) {
   var endpoint = url.parse(pmctl);
-
   if (!endpoint.protocol) {
-    return client.request(pmctl, cmd, callback);
+    pmctl = path.resolve(pmctl);
+
+    if (fs.existsSync(pmctl)) {
+      pmctl = 'http+unix://' + pmctl;
+      remoteHttpRequest(pmctl, cmd, callback);
```

Comment:
I ended up doing pretty much exactly this in 69a438c on my branch if you want to just cherry-pick that into your branch as a fixup.